apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.doc.ts

```diff
@@ -3,13 +3,28 @@ import { Platforms } from '../../../interfaces/Platforms';
 import { TPlatformPageProps } from '@uifabric/example-app-base/lib/index2';
 
 const related = require('!raw-loader!@uifabric/fabric-website/src/pages/Controls/ButtonPage/docs/ButtonRelated.md') as string;
+const componentUrl = 'https://github.com/OfficeDev/office-ui-fabric-react/tree/master/apps/fabric-website/src/pages/Controls/ButtonPage';
 
 export const ButtonPageProps = (disabled: boolean, checked: boolean): TPlatformPageProps<Platforms> => {
   const externalProps = ExternalProps({ areButtonsDisabled: disabled, areButtonsChecked: checked });
   return {
     web: {
       ...(externalProps as any),
       related
+    },
+    ios: {
+      title: 'Button',
+      fileNamePrefix: 'Button',
```

Comment:
`fileNamePrefix` is only needed if the page title differs from the prefix on the markdown files. It's used when we long page title but don't want really long file names.

And this is a nit, but `title: 'Button'` on both objects can be replaced by adding `title="Button"` prop to the `Page` component in `ButtonPage`.
```suggestion
```